### ISSUE TITLE
Request for inverse property of 'fundedItem'

### ISSUE THREAD

**Issue by ashepherd** [schemaorg/schemaorg#2258]
I find that often the FundingAgency issuing a Grant isn't aware of the CreativeWork(s) that were generated downstream. I propose that we create an inverse property to https://schema.org/fundedItem called 'fundedBy' that will let a CreativeWork express which Grant funded its creation.

```
{
  "@context": {
    "schema": "http://schema.org/",
    "bibo": "http://purl.org/ontology/bibo/",
    "dc": "http://purl.org/dc/elements/1.1/",
    "dcat": "http://www.w3.org/ns/dcat#",
    "dct": "http://purl.org/dc/terms/",
    "dcterms": "http://purl.org/dc/terms/",
    "dctype": "http://purl.org/dc/dcmitype/",
    "eli": "http://data.europa.eu/eli/ontology#",
    "foaf": "http://xmlns.com/foaf/0.1/",
    "owl": "http://www.w3.org/2002/07/owl#",
    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
    "rdfa": "http://www.w3.org/ns/rdfa#",
    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
    "schema": "http://schema.org/",
    "skos": "http://www.w3.org/2004/02/skos/core#",
    "snomed": "http://purl.bioontology.org/ontology/SNOMEDCT/",
    "void": "http://rdfs.org/ns/void#",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
    "xsd1": "hhttp://www.w3.org/2001/XMLSchema#"
  },
  "@id": "fundedBy",
  "@type": "rdf:Property",
  "rdfs:comment": "Indicates a <a class=\"localLink\" href=\"http://schema.org/Grant\">Grant</a> that funded some Thing.",
  "rdfs:label": "fundedBy",
  "schema:domainIncludes": {
    "@id": "Thing"
  },
  "schema:inverseOf": {
    "@id": "fundedItem"
  },
  "schema:rangeIncludes": {
    "@id": "Grant"
  }
}
```

_Originally posted by @agbeltran in https://github.com/schemaorg/schemaorg/issues/383#issuecomment-259062293_

### Dataset ###
```
{
  "@context": "http://schema.org",
  "@type": "Dataset",
  "name": "larval krill pigments",
  "fundedBy": {
    "@type": "MonetaryGrant",
    "name": "GLOBEC: Winter Ecology of Larval Krill: Quantifying their Interaction with the Pack Ice Habitat",
    "url": "https://www.nsf.gov/awardsearch/showAward?AWD_ID=9909933",
    "identifier": "9909933",
    "funder": {
      "@type": "FundingAgency",
      "name": "National Science Foundation",
      "alternateName":  "NSF",
      "sameAs": "https://doi.org/10.13039/100000001"
    }
  }
}
```

### ResearchProject ###
```
{
  "@context": "http://schema.org",
  "@type": "ResearchProject",
  "name": "Biological and Chemical Oceanography Data Management Office",
  "fundedBy": {
    "@type": "MonetaryGrant",
    "name": "GLOBEC: Winter Ecology of Larval Krill: Quantifying their Interaction with the Pack Ice Habitat",
    "url": "https://www.nsf.gov/awardsearch/showAward?AWD_ID=1435578",
    "identifier": "1435578",
    "funder": {
      "@type": "FundingAgency",
      "name": "National Science Foundation",
      "alternateName":  "NSF",
      "sameAs": "https://doi.org/10.13039/100000001"
    }
  }
}
```

**Comment by smrgeoinfo** [comment]
@danbri is there some way to nudge this suggestion along?

**Comment by smrgeoinfo** [comment]
we need the ability to link from a resource to grant supporting its development. Unfortunately foaf:fundedBy is out there,  with this scope note "An organization funding a project or person." so its sameAs sdo:funder.  Best solution still seems to go for sdo:fundedBy as having range sdo:Grant--that's more useful for academic products.

**Comment by ashepherd** [comment]
linking to examples for the inverse property of `fundedItem` https://github.com/schemaorg/schemaorg/pull/2618#issuecomment-771799575

**Comment by stain** [comment]
As #3056 was fixed in 14.0 by adding the inverse `funding` this issue can be closed.
